public/app/plugins/panel/state-timeline/StateTimelinePanel.tsx

```diff
@@ -119,6 +123,40 @@ export const StateTimelinePanel: React.FC<TimelinePanelProps> = ({
               renderTooltip={renderCustomTooltip}
             />
             <OutsideRangePlugin config={config} onChangeTimeRange={onChangeTimeRange} />
+
+            {data.annotations && (
+              <AnnotationsPlugin annotations={data.annotations} config={config} timeZone={timeZone} />
+            )}
+
+            <AnnotationEditorPlugin data={alignedFrame} timeZone={timeZone} config={config}>
```

Comment:
Question: Would we not need to check if annotations creation is enabled like [here](https://github.com/grafana/grafana/blob/main/public/app/plugins/panel/timeseries/TimeSeriesPanel.tsx#L53)?

Reply:
@mdvictor very good catch! Will add that